src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -3587,6 +3622,14 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
                 m_xml.startElementRaw(mxmlTechn);
                 m_xml.tag("natural");
                 m_xml.endElement();
+            } else if (String::fromAscii(SymNames::nameForSymId(sid).ascii()).startsWith(u"handbells")) {
```

Comment:
Why not `} else if (mxmlTechn.startsWith(u"handbells") {`?

Reply:
Because obviously the returned value doesn't starts with this.

Reply:
OK, understood now, after looking at this in a debugger ;-)